README.md

```diff
@@ -21,22 +30,26 @@ The `main()` function is the single thread in the application. It toggles the st
 1. Connect a USB cable between the USB port on the board and the host computer.
 1. Run the following command to build the example project and program the microcontroller flash memory:
 
+    * Mbed CLI 2
+
+    ```bash
+    $ mbed-tools compile -m <TARGET> -t <TOOLCHAIN> --flash
+    ```
+
+    * Mbed CLI 1
+
     ```bash
     $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash
     ```
 
 Your PC may take a few minutes to compile your code.
 
-The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-blinky.bin`.
+The binary is located at: </br>
+**Mbed CLI 2** - `./cmake_build/mbed-os-example-blinky.bin`</br>
```

Comment:
I'd recommend using an unordered list, instead of manual line breaks.

Reply:
Fixed.